config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-ci.yaml

```diff
@@ -1,8 +1,10 @@
 periodics:  
-- interval: 4h
-  name: periodic-cluster-api-provider-aws-test-creds
+- name: periodic-cluster-api-provider-aws-test-creds
```

Comment:
I am not entirely sure if the periodic AWS test creds must run. It verifies if the preset creds can create/delete resources.

Reply:
Let's not remove anything that's already there for now